src/IO/lbsLoadReaderVT.py

```diff
@@ -162,102 +162,102 @@ def read_iteration(self, n_p: int, phase_id: int) -> list:
+                                comm_dict[receiver_obj_id]["send"] = list()
+                                comm_dict[receiver_obj_id]["received"] = list()
 
                             if comm_dict.get(sender_obj_id, None) is None:
                                 comm_dict[sender_obj_id] = dict()
-                                comm_dict[sender_obj_id]['send'] = list()
-                                comm_dict[sender_obj_id]['received'] = list()
+                                comm_dict[sender_obj_id]["send"] = list()
+                                comm_dict[sender_obj_id]["received"] = list()
 
-                            comm_dict[receiver_obj_id]['received'].append(
-                                {'from': from_.get('id', None), 'bytes': bytes_})
+                            comm_dict[receiver_obj_id]["received"].append(
+                                {"from": from_.get("id", None), "bytes": bytes_})
                             if self.verbose:
-                                print(f'{bcolors.BLUE}[LoadReaderVT] Added received Phase:{phase_id}, Comm num: {num}\n'
-                                      f'\t\t\tCommunication entry: {comm}{bcolors.END}')
-                            comm_dict[sender_obj_id]['send'].append({'to': to_.get('id', None), 'bytes': bytes_})
+                                print(f"{bcolors.BLUE}[LoadReaderVT] Added received Phase:{phase_id}, Comm num: {num}\n"
+                                      f"\t\t\tCommunication entry: {comm}{bcolors.END}")
+                            comm_dict[sender_obj_id]["send"].append({"to": to_.get("id", None), "bytes": bytes_})
                             if self.verbose:
-                                print(f'{bcolors.BLUE}[LoadReaderVT] Added sent Phase:{phase_id}, Comm num: {num}\n'
-                                      f'\t\t\tCommunication entry: {comm}{bcolors.END}')
+                                print(f"{bcolors.BLUE}[LoadReaderVT] Added sent Phase:{phase_id}, Comm num: {num}\n"
+                                      f"\t\t\tCommunication entry: {comm}{bcolors.END}")
 
-            for task in phase['tasks']:
-                task_time = task.get('time')
-                task_object_id = task.get('entity').get('id')
+            for task in phase["tasks"]:
+                task_time = task.get("time")
+                task_object_id = task.get("entity").get("id")
 
-                # Update processor if iteration was requested
+                # Update rank if iteration was requested
                 if phase_ids in (phase_id, -1):
                     # Instantiate object with retrieved parameters
                     obj = Object(task_object_id, task_time, node_id)
 
                     # If this iteration was never encoutered initialize proc object
-                    returned_dict.setdefault(phase_id, Processor(node_id))
+                    returned_dict.setdefault(phase_id, Rank(node_id))
```

Comment:
This change is already in develop.

Reply:
Same -- manual resolution of conflict between 2 branches being worked in parallel